src/main/resources/static/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Welcome to SurveyMonkey</title>
```

Comment:
Good call including an index page, however the links do not work. Either we should make a controller for this page (@MiniEinsteinD ) or the th:hrefs should be replaced with just href tags if it's a simple page.